mithril-common/src/messages/certificate_pending.rs

```diff
@@ -62,21 +63,55 @@ impl CertificatePendingMessage {
 
 #[cfg(test)]
 mod tests {
+    #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
Do we want to test the parsing of a previous message with a new node?
We'd probably better add a new golden test where the `beacon` field is optional :thinking:

Reply:
We do not test the parsing of a previous message with a new node. We do the opposite.
We check that a new message (with beacon) can be deserialized into the previous CertificatePendingMessage structure.
We also verify that the message, with and without beacon, can be deserialized int the actual CertificatePendingMessage structure. That ensure that the signer will be compatible with the next version when the beacon will be removed.